### PR TITLE
More videos

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding 2 main talks, and 3 lightning talks.

cc @joshmoore @sbesson @will-moore

### CODE REVIEW

**Comment by joshmoore** [comment]
:+1:

**Comment by pwalczysko** [comment]
Looks good on https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2020/day1/

File: events/ome-community-meeting-2020/day2/index.html [hunk 1]
```diff
@@ -14,7 +14,8 @@
       vid_title2: Next Gen File Formats
```
Comment (sbesson) [inline events/ome-community-meeting-2020/day2/index.html:14]:
```suggestion
      vid_title2: "Next Generation File Formats for BioImaging"
```

**Comment by pwalczysko** [comment]
@joshmoore happy with the title update ?

**Comment by pwalczysko** [comment]
Still builds well, thanks @sbesson

**Comment by joshmoore** [comment]
Definitely

**Pull request merged**